librz/core/cmd_descs/cmd_type.yaml

```diff
@@ -50,6 +50,65 @@ commands:
     args:
       - name: type
         type: RZ_CMD_ARG_TYPE_STRING
+  - name: te
+    summary: List loaded enums
+    subcommands:
+      - name: te
+        cname: type_list_enum
+        summary: List loaded enums / Show enum member
+        modes:
+          - RZ_OUTPUT_MODE_STANDARD
+          - RZ_OUTPUT_MODE_JSON
+          - RZ_OUTPUT_MODE_SDB
+        args:
+          - name: enum
+            type: RZ_CMD_ARG_TYPE_STRING
+            optional: true
+          - name: value
+            type: RZ_CMD_ARG_TYPE_STRING
+            optional: true
+      - name: teb
+        cname: type_enum_bitfield
+        summary: Show enum bitfield
+        args:
+          - name: enum
+            type: RZ_CMD_ARG_TYPE_STRING
+          - name: field
+            type: RZ_CMD_ARG_TYPE_STRING
+      - name: tec
+        cname: type_enum_c
+        summary: Show enum in the C output format
+        args:
+          - name: enum
+            type: RZ_CMD_ARG_TYPE_STRING
+            optional: true
+      - name: ted
```

Comment:
this should probably be a subcommand of `tec` (e.g. `tecn`, because also `?e` has `?en` to avoid newlines).

Reply:
We have the same pattern for `tsd`, `tud`, etc. Do you suggest to convert all of them?